fitbenchmarking/fitbenchmarking
# Improvement of Logging on Fitbenchmarking

#### Description of Work

Fixes #110 

The following changes have been made.
1. Log messages now include the name of the file that the messages have been sent from.
2. Log messages with level WARNING or higher will now display on the console.
3. Format of log messages have been improved. Further improvements will be completed to make the log file more useful.
4. An additional page added on Wiki explaining the logging tool on Python and how to utilise it.

#### Testing Instructions

1. Try running `example_runScripts.py` in the case of no exceptions and observe the log messages displayed on the console and the log file (fitbenchmarking.log).
2. Try runnning 'example_runScripts.py' with different intentional exceptions to trigger WARNING/ERROR log messages and observe the log messages displayed on the console and the log file (fitbenchmarking.log).
3. Proofread the added page on Wiki and check for any improvements.
